music21/musicxml/xmlToM21.py

```diff
@@ -2813,12 +2813,12 @@ def xmlToChord(self, mxNoteList: t.List[ET.Element]) -> chord.ChordBase:
                 sp.replaceSpannedElement(n, c)
             for art in n.articulations:
                 if type(art) in seenArticulations:  # pylint: disable=unidiomatic-typecheck
-                    pass
+                    continue
```

Comment:
Thank you! For Myke's benefit: this was identified as a TODO in https://github.com/cuthbertLab/music21/issues/909#issuecomment-796903557.

Reply:
👍🏼